apps/ejabberd/src/mod_http_client_worker.erl

```diff
@@ -0,0 +1,25 @@
+%%==============================================================================
+%% Copyright 2014 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%==============================================================================
+-module(mod_http_client_worker).
+
+%% API
+-export([start_link/1]).
+
+%%------------------------------------------------------------------------------
+%% API
+
+start_link([Host, Opts]) ->
+    fusco:start_link(Host, Opts).
```

Comment:
Can we use `fusco` directly and not through this short module? Or is there a plan to add more functionality to this module?